feathr-example/installment-payments_credit_card_balance.ipynb

```diff
@@ -0,0 +1,1073 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Feathr Feature Store on Home Credit\n",
+    "\n",
+    "This notebook illustrates the use of Feature Store to create a model for home credits. It includes these steps:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prerequisite: Install Feathr\n",
+    "\n",
+    "Install Feathr using pip:\n",
+    "\n",
+    "`pip install -U feathr pandavro scikit-learn`\n",
+    "\n",
+    "Or if you want to use the latest Feathr code from GitHub:\n",
+    "\n",
+    "`pip install -I git+https://github.com/linkedin/feathr.git#subdirectory=feathr_project pandavro scikit-learn`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: feathr in /Users/dasorbit/projects/merlion-feature-store-scenario/.venv/lib/python3.8/site-packages (0.5.1)\n",
```

Comment:
Logs

Reply:
Cleaned up the outputs of the notebooks.